### PR TITLE
Syncronizing version 1.1 changes into gios version 1.0.7

### LINKED ISSUE #37: integrate 1.1.0 changes
I've pulled the latest release into our fork on `upstream-changes-1-19-2016` could you test these out?

### PR DESCRIPTION
**Pull request by chasethenag420**
Closes #37

### CODE REVIEW

**Comment by rraub** [comment]
I can't get the edit lock to work on this version, did something change that breaks this?

**Comment by chasethenag420** [comment]
I tested once again and I see edit lock is working. By the way latest code is on nmyla-develop branch not on upstream-changes-1-19-2016 if you are taking code base from that repo.

File: src/js/fee.js [hunk 1]
```diff
@@ -125,10 +125,7 @@
 
       if (content !== 'raw') {
         returnContent = returnContent.replace(/<p>(?:<br ?\/?>|\u00a0|\uFEFF| )*<\/p>/g, '<p>&nbsp;</p>');
-        returnContent = returnContent.replace(/<p>\[/g, '[');
-        returnContent = returnContent.replace(/\]<\/p>/g, ']');
-        returnContent = returnContent.replace(/<br \/>/g, '');
-        returnContent = returnContent.replace(/<p>&nbsp;<\/p>/g, '<br />');
+        returnContent = window.switchEditors.pre_wpautop(returnContent);
```
Comment (rraub) [inline src/js/fee.js:128]: we should confirm what pre_wpautop is doing and if we want that behavior or not

**Comment by rraub** [comment]
Also FYI the edit locks wont work if you are the same user, that was why I thought they were not working.

File: src/js/fee.js (hunk 1 above)
Comment (chasethenag420) [inline src/js/fee.js:128]: This removes p and br tags which are not inside pre or script tags as similar to wpautop. But prewpautop focuses only on reduced set of elements when compared to wpautop. It prettifies content by removing extra spaces and adding newline characters.

Comment (rraub) [inline src/js/fee.js:128]: ok, so it sounds like its an improvement on the existing filtering. The filtering will always be an issue we have to keep an eye on.

**Pull request merged**